internal/cli/input.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/auth0/auth0-cli/internal/prompt"
+	"github.com/spf13/cobra"
+)
+
+type CommandInput interface {
```

Comment:
Small nit: this can be made private since we don't expect to use it outside the cli package.